Makefile

```diff
@@ -124,7 +124,9 @@ GIT_VERSION: .git
 envoy/SOURCE_VERSION: .git
 	git rev-parse HEAD >envoy/SOURCE_VERSION
 
-docker-image: clean GIT_VERSION envoy/SOURCE_VERSION
+docker-image: clean docker-image
```

Comment:
Is this intentionally depending on itself, or did you mean to depend on `docker-image-dev`?

Reply:
Still WIP ;-)